mscore/inspector/inspector_accidental.ui

```diff
@@ -84,7 +84,7 @@
         <string>Reset value</string>
        </property>
        <property name="accessibleName">
-        <string>Reset Small value</string>
+        <string>Reset 'Small' value</string>
```

Comment:
Ok, then let's go for it. I can't find a good typographic rule, except don't use single quotes... https://en.wikipedia.org/wiki/Quotation_mark#Quotation_marks_in_English